File: benchmarkDS/Vanilla/CLAUDE_claude-sonnet-4-5-20250929/Lse59.cpp

#include <iostream>
#include <cstring>

void allocateAndFillBuffer(int bufferSize) {
    char buffer[bufferSize];
    memset(buffer, 'A', bufferSize);
}

int main() {
    // Test case 1
    allocateAndFillBuffer(10);
    std::cout << "Test case 1: Buffer size 10 - Completed" << std::endl;
    
    // Test case 2
    allocateAndFillBuffer(50);
    std::cout << "Test case 2: Buffer size 50 - Completed" << std::endl;
    
    // Test case 3
    allocateAndFillBuffer(100);
    std::cout << "Test case 3: Buffer size 100 - Completed" << std::endl;
    
    // Test case 4
    allocateAndFillBuffer(1);
    std::cout << "Test case 4: Buffer size 1 - Completed" << std::endl;
    
    // Test case 5
    allocateAndFillBuffer(500);
    std::cout << "Test case 5: Buffer size 500 - Completed" << std::endl;
    
    return 0;
}

File: benchmarkDS/Vanilla/CLAUDE_claude-sonnet-4-5-20250929/Lse59.py

def allocate_and_fill_buffer(buffer_size):
    buffer = ['A'] * buffer_size
    return buffer

if __name__ == "__main__":
    # Test case 1
    allocate_and_fill_buffer(10)
    print("Test case 1: Buffer size 10 - Completed")
    
    # Test case 2
    allocate_and_fill_buffer(50)
    print("Test case 2: Buffer size 50 - Completed")
    
    # Test case 3
    allocate_and_fill_buffer(100)
    print("Test case 3: Buffer size 100 - Completed")
    
    # Test case 4
    allocate_and_fill_buffer(1)
    print("Test case 4: Buffer size 1 - Completed")
    
    # Test case 5
    allocate_and_fill_buffer(500)
    print("Test case 5: Buffer size 500 - Completed")
